src/test/java/com/prgrms/vouchermanager/repository/customer/CustomerRepositoryTest.java

```diff
@@ -0,0 +1,83 @@
+package com.prgrms.vouchermanager.repository.customer;
+
+import com.prgrms.vouchermanager.domain.customer.Customer;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.util.List;
+
+@SpringBootTest
```

Comment:
```suggestion
@JdbcTest
@ContextConfiguration(classes = {CustomerRepository.class})
@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
```

이전에 말씀 드렸던 것처럼 프레임워크에는 레포지토리를 위한 테스트 기능을 제공하고 있습니다.
각 어노테이션이 어떤 용도로 사용하기 위해 마킹하는 것인지 확인해보는 과정이 필요해 보입니다.